devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -48,6 +48,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000,0001,0003,0004,0005,0009,000A,0020,0101", outClusters: "000A,0019", manufacturer: "ASSA ABLOY iRevo", model: "iZBModule01", deviceJoinName: "Yale Door Lock" //Yale Locks (YDF30/40, YMF30/40) with old firmware (v.9.0)
 		fingerprint profileId: "0104", inClusters: "0000,0001,0003,0004,0005,0009,000A,0020,0101", outClusters: "000A,0019", manufacturer: "ASSA ABLOY iRevo", model: "c700000202", deviceJoinName: "Yale Door Lock" //Yale Fingerprint Lock YDF40
 		fingerprint profileId: "0104", inClusters: "0000,0001,0003,0004,0005,0009,000A,0020,0101", outClusters: "000A,0019", manufacturer: "ASSA ABLOY iRevo", model: "0700000001", deviceJoinName: "Yale Door Lock" //Yale Fingerprint Lock YMF40
+        fingerprint profileId: "0104", inClusters: "0000,0001,0003,0101", outClusters: "0000,0001,0003,0101", manufacturer: "Datek", model: "ID Lock 150", deviceJoinName: "ID Lock 150" //ID Lock 150 Zigbee Module
```

Comment:
Please add the following changes:

- change deviceJoinName to "Datek Door Lock"

- fix spacing - use 1 tab instead of 4 spaces

Reply:
Name of door lock is ID Lock 150. Datek is manufacturer of adding module only.

Reply:
Name has been changed to: ID Lock Door Lock
Spacing is fixed.

Reply:
See fork at: https://github.com/AppIdlock/SmartThingsPublic